modules/internal/ChapelLocale.chpl

```diff
@@ -616,6 +616,11 @@ module ChapelLocale {
     return blockedTasks;
   }
 
+  pragma "no doc"
+  proc <(a: locale, b: locale) {
```

Comment:
Oog... this seems unfortunate to me.  Whether or not it's doc'd, it'll be available to users -- is that what we want?  If users find out (or stumble into) the fact that '<' is supported for locales, they'll want to know why <=, >, >= aren't.  I assume that this is to support the sorted() call above?  I'm guessing we don't have a way to pass a comparator record/function into the sorted() iterator yet?  (tagging @ben-albrecht since he implemented that capability for normal sorts).